Makefile

```diff
@@ -136,14 +139,16 @@ endif
 	make fetch-sample-data
 	dotnet tool restore
 
-environment-ui: environment
+environment-ui:
```

Comment:
environment-ui is now independent from environment. since we are using the encryption nuget package now there's no need to build the c++ as part of the UI project dependencies and so we dont need all of its tooling